mautic/mautic
# Add support to auto fill data for others field types

closes #6286 
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added support to auto fill form data for others kind of fields.
We noticed: url, website, date, date time was not supported.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create form with email, url, website, date, date time fields
2. All fields match to contact field and enable Auto fill data for each of them

![image](https://user-images.githubusercontent.com/462477/86199701-0fd11480-bb5b-11ea-8ef2-5afab9cef8d7.png)

3. Try submit form
4. Reload form. You should noticed, just email as filled field.

#### Steps to test this PR:
1. Load up [this PR](https://m3.mautibox.com)
2. After repeat all steps you should see all fields auto filled 

![image](https://user-images.githubusercontent.com/462477/86199769-473fc100-bb5b-11ea-9f75-8117a08b710f.png)
